content/en/docs/setup/install/operator/index.md

```diff
@@ -75,25 +73,210 @@ accessible to `istioctl` by using this command:
+Another customization is to select different namespaces for features and components. The following is an example
+of installation namespace customization:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  defaultNamespace: istio-system
+  security:
+    namespace: istio-security
+    components:
+      citadel:
+        namespace: istio-citadel
+{{< /text >}}
+
+Applying this file will cause the default profile to be applied, with components being installed into the following
+namespaces:
+
+- The Citadel component is installed into `istio-citadel` namespace
+- All other components in the security feature installed into `istio-security` namespace
+- Remaining Istio components installed into istio-system namespace
+
+## Customize Kubernetes settings using the `IstioControlPlane` API
+
+The `IstioControlPlane` API allows each component's Kubernetes settings to be customized in a consistent way.
+
+### Identify the feature or component settings
+
+Each component has a [`KubernetesResourceSpec`](/docs/reference/config/istio.operator.v1alpha12.pb/#KubernetesResourcesSpec),
+which allows the following settings to be changed. Use this list to identify the setting to customize:
+
+1. [Resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container)
+1. [Readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)
+1. [Replica count](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+1. [`HorizontalPodAutoscaler`](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+1. [`PodDisruptionBudget`](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#how-disruption-budgets-work)
+1. [Pod annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+1. [Service annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+1. [`ImagePullPolicy`](https://kubernetes.io/docs/concepts/containers/images/)
+1. [Priority class name](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass)
+1. [Node selector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector)
+1. [Affinity and anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
+
+All of these Kubernetes settings use the Kubernetes API definitions, so [Kubernetes documentation](https://kubernetes.io/docs/concepts/) can be used for reference.
+
+### Configure the feature or component
+
+After you identify the name of the feature or component from the previous list, you can use the `IstioControlPlane` API
+to modify the default values using a configuration overlay file.
+
+For example, use this overlay file ([`samples/pilot-k8s.yaml`](https://github.com/istio/operator/blob/release-1.3/samples/pilot-k8s.yaml))
```

Comment:
see @geeknoid 's PR for a soltuion to this problem: https://github.com/istio/istio.io/pull/4952

Reply:
Done.